examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
Tried with `#include <SDL2/SDL.h>` and broke... Quite strange... will take a look.